gdpr_assist/management/commands/anonymise_db.py

```diff
@@ -19,13 +75,29 @@ def add_arguments(self, parser):
             default=True,
             help="Tells Django to NOT prompt the user for input of any kind.",
         )
+        parser.add_argument(
+            "--data-strategies",
+            "--strategies",
+            "-s",
+            dest="strategies",
+            default="anonymisable:anonymise, unanonymisable:retain, untagged:retain",
+            help="Configures the anonymisation strategies that will be applied.",
+        )
 
     def handle(self, *args, **options):
         if not app_settings.GDPR_CAN_ANONYMISE_DATABASE:
             raise ValueError("Database anonymisation is not enabled")
+
         interactive = options["interactive"]
+        strategies = StrategyHelper.parse(options["strategies"])
 
         if interactive:  # pragma: no cover
+            self.stdout.write(
+                "The following data anonymisation strategies will be applied:"
+            )
+            for category, strategy in strategies.items():
+                self.stdout.write(" - {category} data: {strategy.upper()}")
```

Comment:
This messaging (and the way it formats on-screen) can probably be improved.  I had an idea to potentially ask the user to confirm the intended strategy for each category, instead of / as well as simply writing `yes` to commence the anonymisation in interactive mode.